### PR TITLE
One less dependency!

### PR DESCRIPTION
**Pull request by FelipeIzolan**
Hi, I removed axios and replaced it with http. (fetch.ts, api.ts)
and I change somethings...

### CODE REVIEW

File: src/api.ts [hunk 1]
```diff
@@ -1,8 +1,9 @@
-import axios from 'axios';
+import fetch from './fetch';
```
Comment (Balastrong) [inline src/api.ts:1]: Is there any particular reason why reimplementing `fetch` and not using the native [`fetch`](https://developer.mozilla.org/en-US/docs/Web/API/fetch) method?

File: src/uti.ts [hunk 2]
```diff
@@ -19,22 +19,17 @@ export const getGames = async (
...
-    if (games.length >= amount) {
-      return games.slice(0, amount);
+    if (games.length < amount) {
+      const gamesInArchive = await getChessComGames(archive);
+      games = [...games, ...gamesInArchive.reverse()];
     }
   }
 
-  return games;
+  return games.slice(0, amount);
```
Comment (Balastrong) [inline src/uti.ts]: This actually changes the logic, we previously looked for max 5 archives and return early, now you loop through all archives in any case.

Is there a particular reason for this change?

**Review by Balastrong — CHANGES_REQUESTED** [review]
Thank you for the PR!

There are some interesting changes but I'd like to get more insights of your reasons behind them before approving :)

File: src/api.ts (hunk 1 above)
Comment (FelipeIzolan) [reply src/api.ts:1]: because native fetch not is in node LTS version. (only 17.5^)

File: src/uti.ts (hunk 2 above)
Comment (FelipeIzolan) [reply src/uti.ts]: I forgot to break the loop XD
```javascript
for (const archive of archives) {
   if (games.length < amount) {
      const gamesInArchive = await getChessComGames(archive);
      games = [...games, ...gamesInArchive.reverse()];
   } else { break }
}
```

Comment (Balastrong) [reply src/uti.ts]: Looks better now, thanks :)

**Review by Balastrong — APPROVED** [review]

**Comment by Balastrong** [comment]
LGTM, thanks for the PR! :)

**Pull request merged**